src/diffusers/models/transformers/latte_transformer_3d.py

```diff
@@ -0,0 +1,379 @@
+import torch
+from torch import nn
+
+from dataclasses import dataclass
+from dataclasses import dataclass
+from einops import rearrange, repeat
+from typing import Any, Dict, Optional
+from ...configuration_utils import ConfigMixin, register_to_config
+from ...utils import BaseOutput
+from ..attention import BasicTransformerBlock
+from ..embeddings import PatchEmbed
+from ..modeling_utils import ModelMixin
+from ..normalization import AdaLayerNormSingle
+from ...models.embeddings import (
+    get_1d_sincos_pos_embed_from_grid, 
+    PixArtAlphaTextProjection)
+
+
+@dataclass
+class Transformer3DModelOutput(BaseOutput):
+    sample: torch.FloatTensor
```

Comment:
I think it makes sense to move this to: https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/modeling_outputs.py

Not strong opinions though.

Reply:
Thanks. I moved it to modeling_outputs.